src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,58 @@
+.watch {
+  position: relative;
+  @include size($size);
+  border: 20px solid $main-color;
+  border-radius: 35%;
+  box-shadow: inset 0 0 20px $secondary-color;
+  background-image: url(/image/watch_number.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+
+  &::before {
+    position: absolute;
+    left: 50%;
+    top: calc(50% - 5px);
+    content: "";
+    width: 40%;
+    height: 10px;
+    background-color: $secondary-color;
+    border-radius: 5px;
+    animation: time 60s linear infinite;
+    transform-origin: left;
+  }
+
+  &::after {
+    position: absolute;
+    left: 50%;
+    top: calc(50% - 5px);
+    content: "";
+    width: 30%;
+    height: 10px;
+    background-color: $secondary-color;
+    border-radius: 5px;
+    animation: time 3600s linear infinite;
+    transform-origin: left;
+  }
```

Comment:
Nice work!

Just suggesting for pseudo elements styles create mixin with parameters which different.
But a lot of styles are same.